### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by mespitaleta1**
Hi David. I send to you this pull request to check the issue about the responsive of the website. Please check it, I will waiting for your comments. Thank you.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -53,29 +57,31 @@ body {
...
+  min-height: 250px;
   box-shadow: 18px -17px 0px 1px;
+  flex-basis: 50%;
+  object-fit: cover;
   color: #f1c40f;
 }
 
 .autor_info {
   line-height: 25px;
+  flex-basis: 90%;
```
Comment (Newville23) [inline style.css]: is this necessary?

File: style.css [hunk 2]
```diff
@@ -110,30 +114,44 @@ h2 {
 .year-book {
   text-align: center;
   align-self: center;
-  margin: 20px 12px;
+  margin: 20px 12px 5px;
+}
+
+.year-book {
```
Comment (Newville23) [inline style.css:118]: would you like to add this to one single class?

File: style.css [hunk 3]
```diff
@@ -151,3 +169,48 @@ h2 {
...
+  .menu_bar_link {
+    padding-right: 40px;
+  }
+
+  .wrapper_1 {
+    padding-top: 150px;
+  }
+
+  .title {
+    font-size: x-large;
```
Comment (Newville23) [inline style.css]: specify this in units :)

**Review by Newville23 — CHANGES_REQUESTED** [review]

File: style.css (hunk 2 above)
Comment (mespitaleta1) [reply style.css:118]: yes, I see the problem here. As we talk I will create a new class to wrap both elements in one rule to affect them and leave the class "year-book" with his own margin.

File: style.css (hunk 1 above)
Comment (mespitaleta1) [reply style.css]: these values were rectified and modified. thanks

File: style.css (hunk 3 above)
Comment (mespitaleta1) [reply style.css]: Done!

**Pull request merged**